hles/extraction/src/test/scala/org/broadinstitute/monster/dap/CslbExtractionPipelineBuilderIntegrationSpec.scala

```diff
@@ -0,0 +1,73 @@
+package org.broadinstitute.monster.dap
```

Comment:
I (somewhat selfishly?) like that these are in just `test` now, not `it`, it should play well with Intellij's tooling better.

Reply:
the folder seems to affect `sbt` generally, which is why i made the move